src/app/src/pages/Draw.js

```diff
@@ -8,16 +8,16 @@ import Sidebar from '../components/Sidebar';
 
 export default function Draw() {
     return (
-        <Flex>
-            <Sidebar />
-            <Box flex={1} position='relative'>
-                <Map>
-                    <DrawContextProvider>
+        <DrawContextProvider>
```

Comment:
Not sure if this would cause the map to re-render excessively, hasn't done so in my testing.

Reply:
Apparently this is because of the `fitBounds` call. That `useEffect` is firing more often than it should. Will take a look at it.

https://github.com/azavea/iow-boundary-tool/blob/dab9d760620ac7dd4973f26178c310d97bf84909/src/app/src/components/DrawTools/useEditingPolygon.js#L104

Reply:
[e616180](https://github.com/azavea/iow-boundary-tool/pull/156/commits/e6161800495f68b0800f2fe879473b4eec37c39d)